tasks/main.yml

```diff
@@ -23,6 +23,9 @@
 - import_tasks: django.yml
   tags: [openwisp2, django]
 
+- import_tasks: freeradius.yml
+  tags: [openwisp2, freeradius]
```

Comment:
can you add a var to disable this? In case freeradius is installed on a separate server.